services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -4168,6 +4175,10 @@ public StoreMetaValue getMetaStoreValue(StoreMetaKey metaKey, String storeName)
 
   @Override
   public Set<Integer> getInUseValueSchemaIds(String clusterName, String storeName) {
+    if (isParent()) {
```

Comment:
I'm not sure if this work-around is good enough. This means the un-used schemas will be deleted in child fabrics but still linger around in the parent which will still be visible on Nuage right?

Reply:
emm no. It wil still be deleted in parent, the fix just skips the query in parent colo.

Reply:
I see, thanks for the explanation